analyzers/rspec/cs/S6640.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Using unsafe code blocks is security-sensitive",
+  "type": "SECURITY_HOTSPOT",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "60min"
+  },
+  "tags": [],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-6640",
+  "sqKey": "S6640",
+  "scope": "All",
+  "quickfix": "infeasible"
```

Comment:
Your RSpec change was merged by me. You may want to run the update script again against RSpec master, just to make sure everything is as expected